traefik.go

```diff
@@ -180,3 +194,38 @@ func run(traefikConfiguration *TraefikConfiguration) {
 	defer server.Close()
 	log.Info("Shutting down")
 }
+
+// CreateKvSource creates KvSource
+// TLS support is enable for Consul and ects backends
+func CreateKvSource(traefikConfiguration *TraefikConfiguration) (*staert.KvSource, error) {
+	var kv *staert.KvSource
+	var store store.Store
+	var err error
+
+	if traefikConfiguration.Consul != nil {
```

Comment:
nit: maybe using a `switch` would be a little better (not sure but :stuck_out_tongue_closed_eyes:)